dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchResults.vue

```diff
@@ -55,15 +55,22 @@
       </Column>
     </DataTable>
     <div class="d-center-div text-center px-7 py-4" v-else>
-      <p class="font-medium text-xl">Sorry! Your search didn't return any results.</p>
-      <p class="font-medium">Try again please!</p>
+      <p class="font-medium text-xl">We're sorry, but your search did not return any results.</p>
```

Comment:
The font of the text should be aligned. In the figma both lines have the same font.
![image](https://github.com/d-fine/Dataland/assets/104447806/43a45503-770f-4dec-bf24-da2e232ce03b)

Reply:
I don't have a strong opinion on that. However, I thought it would increase the readability when both sentences are not displayed with exactly the same font sizes. Nevertheless, adapted this.

Reply:
Ok